identity-android/src/main/java/com/android/identity/android/mdoc/deviceretrieval/VerificationHelper.java

```diff
@@ -1179,8 +1180,8 @@ KeyPair getEReaderKeyPair() {
      *
      * @return the curve for the keys used for session encryption.
      */
-    public @SecureArea.EcCurve
-    int getEReaderKeyCurve() {
+    public
+    EcCurve getEReaderKeyCurve() {
```

Comment:
syntax nit: `public` modifier can live in the same line as the rest of the method signature

Reply:
OK.